main/mqtt.c

```diff
@@ -291,6 +292,10 @@ int mqtt_connect(const char *host, uint16_t port, const char *client_id,
         .cert_pem = server_cert,
         .client_cert_pem = client_cert,
         .client_key_pem = client_key,
+        .lwt_topic = status_topic,
+        .lwt_msg = "Offline",
```

Comment:
The LWT payload should also be received as a parameter (which may be hard-coded in `ble2mqtt.c`).